<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapEditor</class>
 <widget class="QMainWindow" name="mapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mapEditorGraphicsView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>561</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QPushButton" name="dirtButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Terre</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="spawnButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Spawn</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cloudButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>nuage</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="lavaButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Lave</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="brickButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>brique</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="endLevelButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Fin niveau</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="grassButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>herbe</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>pushButton_6</zorder>
    <zorder>dirtButton</zorder>
    <zorder>spawnButton</zorder>
    <zorder>cloudButton</zorder>
    <zorder>lavaButton</zorder>
    <zorder>brickButton</zorder>
    <zorder>endLevelButton</zorder>
    <zorder>grassButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
